content/webapp/components/AudioPlayerNew/AudioPlayer.PlayRate.tsx

```diff
@@ -181,15 +180,13 @@ const PlayRate: FunctionComponent<PlayRateProps> = ({
                   onClick={() => updatePlaybackRate(speed)}
                 >
                   <span style={{ display: 'flex', alignItems: 'center' }}>
-                    {speed}x
+                    <span style={{ minWidth: '4em' }}>{speed}x</span>
```

Comment:
In order to make sure the checkmark was always in the same place no matter how many characters there were in the preceding text (e.g. 1x vs 0.5x), it was a toss up between putting a fixed width on the container, or specifying a min-width here. I don't _love_ either of them, but I disliked this one less

Reply:
I trust your CSS gut more than mine and also don't think there's really a wrong answer when it's really unlikely to encounter longer strings.